php/deepgram_transcribe_audio/index.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+require 'vendor/autoload.php';
+
+use GuzzleHttp\Client;
+
+return function($req, $res) {
+    try {
+        $payload = \json_decode($req['payload'], true);
+        if (!isset($payload['fileUrl']) || empty($fileUrl = \trim($payload['fileUrl']))) {
+            throw new \Exception('Please provide valid file url.');
+        }
+        if (
+            !isset($payload['variables']['deepgramApiSecretKey']) || 
+            empty($deepgramSecretApiKey = \trim($payload['variables']['deepgramApiSecretKey']))
+        ) {
+            throw new \Exception('Please provide valid deepgram api secret key under variables.');
+        }
+
+        $extension = pathinfo($fileUrl, PATHINFO_EXTENSION);
+
+        if ($extension !== 'wav') {
+            throw new \Exception('Please provide valid file url extension. This should be wav.');
+        }
```

Comment:
Deepgrams supports more than just wav files don't they?

Reply:
I thought, it's should be only .wav format like in Issue task described. [Issue](https://github.com/appwrite/appwrite/issues/4489)

"The payload must accept a WAV file URL, and the response from Deepgramm should be returned."

If I misunderstood, no problem i can remove this condition

Reply:
ok sure. fair enough